### PR TITLE
feat(react-query): clarify cli command explanations

### PR DESCRIPTION
**Pull request by gwansikk**
# Overview

image

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: packages/react-query/src/scripts/cli.ts [hunk 1]
```diff
@@ -11,22 +11,22 @@ const program = new Command()
...
-  .description('Check the compatibility status of the current version')
+  .description('Checks compatibility with the currently used version of @tanstack/react-query')
   .action(() => statusAction())
 
 program
   .command('switch')
-  .description('Switch to a specific version of @suspensive/react-query')
-  .argument('<version>', 'Switches the version of @suspensive/react-query')
+  .description('Switch the exports to use the specified version of  @suspensive/react-query interface')
+  .argument('<version>', 'Switch the exports to use the specified version of  @suspensive/react-query interface')
```
Comment (manudeli) [inline packages/react-query/src/scripts/cli.ts:24]:
```suggestion
  .argument('<version>', 'Switch the exports of @suspensive/react-query to use compatible Suspensive interfaces for @tanstack/react-query@<version>')
```

File: packages/react-query/src/scripts/cli.ts [hunk 2]
```diff
@@ -11,22 +11,22 @@ const program = new Command()
...
   .action((version) => switchAction(version))
 
 program
   .command('fix')
-  .description('Fix the compatibility issues')
+  .description('Automatically switches to the compatible interface for @tanstack/react-query')
```
Comment (manudeli) [inline packages/react-query/src/scripts/cli.ts:29]:
```suggestion
  .description('Automatically switch @suspensive/react-query exports to use compatible Suspensive interfaces for @tanstack/react-query')
```

**Review by manudeli — COMMENTED** [review]
Update docs please after please

**Comment by gwansikk** [comment]
> Update docs please after please

Could you please explain what you mean in more detail?

- The documentation update was planned for a new PR after this one. Should I include it in this PR instead?

**Comment by manudeli** [comment]
> The documentation update was planned for a new PR after this one. Should I include it in this PR instead?

I think that It's up to your decision

**Comment by gwansikk** [comment]
> I think that It's up to your decision

I have updated the documentation and included it in this PR!

File: docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx [hunk 3]
```diff
@@ -8,13 +8,13 @@ From version 2.2.0 onwards, @tanstack/react-query supports both v4 and v5. To su
 
 </Callout>
 
-@suspensive/react-query internally uses @suspensive/react-query-4 and @suspensive/react-query-5, depending on the version of @tanstack/react-query detected in your environment. It automatically provides a compatible interface.
+@suspensive/react-query is composed of @suspensive/react-query-4 and @suspensive/react-query-5, which provide interfaces corresponding to the version of @tanstack/react-query. It detects the installed version of @tanstack/react-query in the user's environment and automatically provides the compatible interface.
 
 ## Command-Line Interface (CLI)
 
-The Command-Line Interface (CLI) of @suspensive/react-query is a tool to handle compatibility issues with @tanstack/react-query. You can use commands to check compatibility and switch to the compatible interface of @suspensive/react-query.
+The Command-Line Interface (CLI) of @suspensive/react-query is a tool to resolve compatibility issues with @tanstack/react-query. It allows you to check the compatibility with the installed version of @tanstack/react-query and switch the interface of @suspensive/react-query to a compatible one via commands.
```
Comment (manudeli) [inline docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx:15]:
```suggestion
The Command-Line Interface (CLI) of @suspensive/react-query is a tool to match compatibility with @tanstack/react-query. It allows you to check the compatibility with the installed version of @tanstack/react-query and switch the interface of @suspensive/react-query to a compatible one via commands.
```

File: docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx [hunk 4]
```diff
@@ -72,35 +72,34 @@ npx suspensive-react-query --help
...
+Checks the compatibility of the currently installed version of @tanstack/react-query with @suspensive/react-query and displays the compatibility status.
 
 ```bash
 npx suspensive-react-query status
 ```
 
-For instance, if using @suspensive/react-query@2.9.0 and @tanstack/react-query@5, the result might look like this:
+For example, using @suspensive/react-query@2.12.1 and @tanstack/react-query@5.51.21, the result might be.
+
+You can check the compatibility status with status, and see which interfaces are available in your environment in the available interfaces section.
```
Comment (manudeli) [inline docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx:83]:
```suggestion
You can check the compatibility status with `npx suspensive-react-query status`, and see which interfaces are available in your environment in the available interfaces section.
```

**Review by manudeli — COMMENTED** [review]
Almost done! 🙇🙇🙇

File: docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx (hunk 4 above)
Comment (gwansikk) [reply docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx:83]: I noticed some duplicate content, so I made slight revisions to that section.

```md
```

**Review by manudeli — APPROVED** [review]
So clear 🚀🚀🚀🚀🚀 Cool

**Pull request merged**